geode-core/src/test/java/org/apache/geode/internal/cache/ProxyBucketRegionTest.java

```diff
@@ -32,4 +46,54 @@ public void shouldBeMockable() throws Exception {
 
     assertThat(mockProxyBucketRegion.getBucketAdvisor()).isSameAs(mockBucketAdvisor);
   }
+
+  @Test
+  public void testRecoverFromDisk() throws Exception {
```

Comment:
This test name needs to be more descriptive. It should say what behaviour is under test, what the conditions are, and what the expected behaviour is. Also, an exception is never thrown from this method, so this can be removed.